lib/backend/k8s/resources/workloadendpoint.go

```diff
@@ -84,6 +76,14 @@ func (c *WorkloadEndpointClient) Update(ctx context.Context, kvp *model.KVPair)
 	return kvp, nil
 }
 
+func (c *WorkloadEndpointClient) Update(ctx context.Context, kvp *model.KVPair) (*model.KVPair, error) {
```

Comment:
Without digging too much, the CNI plugin also seems to perform Update the WEP resource, and I am not sure which scenario this would get called.

Reply:
Hmm, I was assuming it would only ever call Create() because it will never have a resourceVersion, and this code gets executed: https://github.com/projectcalico/cni-plugin/blob/master/utils/utils.go#L68

HOWEVER, the more I think about it, the more I think there may be scenarios where Update() gets called.

Let's perform this logic in both places.